blaze-client/src/test/scala/org/http4s/client/blaze/MaxConnectionsInPoolSpec.scala

```diff
@@ -26,8 +28,32 @@ class MaxConnectionsInPoolSpec extends Http4sSpec {
     }
   }
 
+  "Blaze Pooled Http1 Client" should {
+    "Behave and not deadlock" in {
```

Comment:
I love having a test like this, but we have problems with external resources on Travis CI.  What if we created a service that responds after a small, random amount of latency, and then bound that service to five local ports?